CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
```

Comment:
Let's use the contributing guide we specify here as the base template and then add the specifics of this repo to that.
https://github.com/appwrite/appwrite/issues/2118

Reply:
Done 👍